src/workerd/api/node/tests/buffer-nodejs-test.js

```diff
@@ -5792,3 +5793,104 @@ export const transcodeTest = {
     }
   }
 };
+
+// Tests are taken from Node.js
+// https://github.com/nodejs/node/blob/main/test/parallel/test-file.js
```

Comment:
it's pretty minor, but when linking to a gh repo I always prefer permalinks so that if there are changes the link is still valid:
```suggestion
// https://github.com/nodejs/node/blob/a4f609fa/test/parallel/test-file.js
```

Reply:
Tip: hit 'y' on github to get the SHA based URL